### ISSUE TITLE
Call Docker Compose in-process instead of as an external process

### ISSUE THREAD

**Issue by arthur-tacca** [eywalker/nvidia-docker-compose#12]
Docker Compose is written in Python and hosted on PyPI, so instead of calling it with the subprocess module you could call it directly. This has two benefits: first, the user doesn't need to manually install docker-compose separately, and second, signals (including Ctrl+C) will be handled better.

These are the changes needed:
 * Add `"docker-compose"` to `install_requires` in `setup.py`. 
 * Add this to the import list: `from compose.cli.main import main as compose_main`
 * Replace the `subprocess` call (including the `try`/`except` block) with this:

```
sys.argv[:] = ['docker-compose', '-f', args.output.name] + extras
compose_main()
```

**Comment by eywalker** [comment]
@arthur-tacca: Brilliant suggestion! Since it looks like you have it pretty much all figured out, if you make a PR incorporating the suggested changes, I'd be very happy to review and accept.

**Comment by eywalker** [comment]
Issue addressed and fixed with PR #13.
